src/ui-components/cards/summary-card-footer.component.tsx

```diff
@@ -1,15 +1,19 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import styles from "./summary-card-footer.css";
+import { Trans } from "react-i18next";
 
 export default function SummaryCardFooter(props: SummaryCardFooterProps) {
   if (!props.linkTo) {
     return (
       <div className={styles.footer}>
-        <p className="omrs-bold">See all</p>
+        <p className="omrs-bold">
+          <Trans i18nKey="See all">See all</Trans>
```

Comment:
I noticed that you used the `t` function in some places while using the `Trans` component in others but I don't see much difference in context. Wouldn't `t` been enough in all instances? Could you clarify me on your reason? Not an expert with the `react-i18next` so I'm speaking based on whats on the documentation here https://react.i18next.com/latest/trans-component.

Reply:
This is perhaps not the best use case for the `Trans` component (most of this code has been refactored locally). Basically, the `useTranslation` hook can only be used inside function components. There are occurrences in our code where helper functions have been written outside of function components and therefore `useTranslation` cannot be used. It is in these situations where I make use of the `Trans` component. 

A brief aside, you'll notice that the translation keys specified here are not specified in any standard format. I've refactored them to try as much as possible to use camelCase. I think it'd be easier going forward if we stuck to that.

Reply:
Ok. Thanks for the clarification.